escape_roomba/thread_channel.py

```diff
@@ -158,33 +174,64 @@ async def async_maybe_create_from_origin(client, channel_id, message_id):
             number += 1
             name = f'{basic_name}-{number}'
 
+        # Place the new channel after the origin message's channel.
+        position = message.channel.position + 1
```

Comment:
Interesting. I thought your plan was to place the channels in a special "threads" area, but I actually think I like this system better. It will be interesting to see how it pans out in practical use.

Reply:
Yep!

Reply:
We shall see indeed!